src/argilla/training/openai.py

```diff
@@ -56,7 +57,13 @@ def __init__(self, *args, **kwargs):
             self._logger.warning("Seed is not supported for OpenAI. Ignoring seed for training.")
 
         if self._model is None:
-            self._model = "curie"
+            self._model = "gpt-3.5-turbo"
+
+        if any([self._model in model for model in OPENAI_LEGACY_MODEL]):
+            self._logger.warning("Using the OpenAI legacy trainer, please use a newer model.")
```

Comment:
Maybe here we could provide a more insightful message to the users

Reply:
Perfect, changed it.